packages/components/src/tools-panel/tools-panel/hook.js

```diff
@@ -21,6 +21,14 @@ export function useToolsPanel( props ) {
 		return cx( styles.ToolsPanel, className );
 	}, [ className ] );
 
+	const isResetting = useRef( false );
+
+	useEffect( () => {
+		if ( isResetting.current ) {
+			isResetting.current = false;
+		}
+	} );
```

Comment:
Without any dependencies listed, this `useEffect` will be executed for every render. Is that intended? If yes, to what purpose (and why would `useEffect` be necessary)?

Reply:
✅ Fixed. Thanks.